src/bp-templates/bp-nouveau/js/buddypress-media.js

```diff
@@ -317,8 +317,6 @@ window.bp = window.bp || {};
 			$( '.groups.messages form#send_group_message_form #whats-new-toolbar .bp-group-messages-attached-gif-container .gif-search-results' ).scroll( this.loadMoreGroupMessagesGif.bind( this ) );
 			groupMessagesToolbarContainer.on( 'click', '.gif-image-remove', this.removeGroupMessagesSelectedGif.bind( this ) );
 
-			$( document ).on( 'click', '.gif-image-container', this.playVideo.bind( this ) );
```

Comment:
@rmmehta This line was added for `gif play pause button fix` so can you verify that is working with removing this line. 
This line added 2 times on the same function so might be it work just confirmation is needed before sending it to QA.